src/templates/apollo.config.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-env node */
+// See https://www.apollographql.com/docs/devtools/apollo-config/
+module.exports = {
+  client: {
+    service: {
+      name: 'my-service',
+      // URL to the GraphQL API
```

Comment:
```suggestion
```
I think the property name is self-explanatory, and it's even more with the supplied example value.